Topics header design implementation

smallfx opened the pull request:
Implements "main page" Topics view 🐯
Also includes (very related) fixes to Paginator component.

stephensprinkle commented:
Looks good overall — a few things to clean up:

- [use `em` for relationally positioned elements + `rem` for globally relationally positioned elements]
- [no margin present on the categories view resulting in cats that overflow]
- [refactor hard coded `px` widths to instead flow with element layout, add padding or margin where necessary]

smallfx commented:
@stephensprinkle Last commit addresses those concerns with special consideration for `vw` vs `rem`/`em` in certain cases where basing the width on the total screen width was more appropriate for layout fluidity

stephensprinkle commented:
I ended up rolling back the flex implementation as it ended up not being as clean of an implementation.
